tests/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -128,7 +128,7 @@ private function filterElementsByName(array $items, string $name): array
         return array_filter(
             $items,
             static function (AbstractAsset $item) use ($name): bool {
-                return $item->getShortestName($item->getNamespaceName()) === $name;
+                return strtolower($item->getName()) === $name;
```

Comment:
Previously, `AbstractAsset::getShortestName()` would call `strtolower()` internally, so we just move it here. We don't add it in the other cases because `Schema::(has|get)(Table|Sequence)()` does that internally.